intel-retail/automated-self-checkout
# feat: docker-compose changes for integration with Developer Toolbox

## PR Checklist



## What are you changing?
#338

## Issue this PR will close

close: #338 

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- Build apps and download models
- Run `make run-pipelines` to start all the containers
- Validate if following containers are up & running - 2 object-detection, 1 classification, 1 ovms, 3 camera-simulator containers.
- Check pipeline & result logs, UI support is not available currently.
- docker-compose can be customized based on instructions in the comment section
- Run `make down-pipelines` to stop & remove all the containers
- make serve-docs, check documentation

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
